StortSpelProjekt/Engine/src/Misc/Window.cpp

```diff
@@ -225,12 +225,12 @@ const HWND* Window::GetHwnd() const
 	return &m_Hwnd;
 }
 
-void Window::SetScreenWidth(int width)
+void Window::SetScreenWidth(const int width)
```

Comment:
Why const? It's not a pointer or a reference, so the source can't be changed anyway..

Reply:
stuff left from trying to make it work dynamically. There was a lot of shenanigans concerning const and windows.